gigascience/gigadb-website
# Use new layout on pages missed on first pass

# Pull request for issue: #1519

This is a pull request for the following functionalities:

* A list of pages missing the update to modern layout were updated:

- http://gigadb.gigasciencejournal.com:9170/lorem
- http://gigadb.gigasciencejournal.com:9170/datasetSubmission/upload
- http://gigadb.gigasciencejournal.com:9170/dataset/1001
- http://gigadb.gigasciencejournal.com:9170/adminFileFormat/create
- http://gigadb.gigasciencejournal.com:9170/curationLog/create/id/5

* A11y issues on those pages have been fixed

## How to test?

* Navigate to the above pages and confirm that they use the modern layout
* Use WAVE extension to run an a11y audit and verify that there are no reported errors

## How have functionalities been implemented?

* Updated the pages to use one of the new layouts
* Updated the page layout following a similar pattern to other pages

## Any issues with implementation?

None

## Any changes to automated tests?

* One test was updated to match an updated link label, e.g. `Download Template File` -> `Download Template Excel File` based on:

![image](https://github.com/gigascience/gigadb-website/assets/143437854/09c0d7d4-a497-486a-8d70-9199a961a318)


## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
